### DISCUSSION TITLE
Grafana vulnerability released

### DISCUSSION THREAD

**Discussion by yomaoni** [Security-Onion-Solutions/securityonion#8943] [Unsupported Versions] [answered]
Not sure if these apply to security onion currently but I saw these and figured I would pass them along.

https://nvd.nist.gov/vuln/detail/CVE-2022-31123
https://nvd.nist.gov/vuln/detail/CVE-2022-31130

**Comment by dougburks** [comment, marked as answer]
These vulnerabilities shouldn't be an issue for us, but we'll plan to go ahead and upgrade Grafana anyway:
https://github.com/Security-Onion-Solutions/securityonion/issues/8950
